play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/internal/workers/PublishArtifactWorkerBase.kt

```diff
@@ -22,10 +22,11 @@ internal abstract class PublishArtifactWorkerBase<T : PublishArtifactWorkerBase.
         return if (config.releaseName != null) {
             config.releaseName
         } else if (parameters.consoleNamesDir.isPresent) {
-            val dir = parameters.consoleNamesDir.get().asFile
-            val file = File(dir, "$track.txt").orNull()
-                    ?: File(dir, RELEASE_NAMES_DEFAULT_NAME).orNull()
-
+            val dir = parameters.consoleNamesDir.get()
+            val file = dir.file("${track.replace(':', '-')}.txt").asFile.orNull()
+                    ?: dir.file("${track.substringAfter(':')}.txt").asFile.orNull()
+                    ?: dir.file("$track.txt").asFile.orNull()
```

Comment:
Shouldn't this be removed entirely? If you don't have a file named `formfactor-track.txt` or `track.txt`, it's still going to crash with this right as it'll try `formfactor:track.txt`. Pretty sure `track.substringAfter(':')` covers `track` as if there's no colon it'll do nothing.

Reply:
Good catch, I’m not sure whether simply calling `.file("file:with:colon.txt")` will crash or return a null instead, but like you said, since `.substringAfter(':')` covers this, we can just remove this line.